cmd/nodecmd/create.go

```diff
@@ -252,15 +298,12 @@ func createNode(_ *cobra.Command, args []string) error {
 	}
 	PrintResults(instanceID, elasticIP, certFilePath, region)
 	ux.Logger.PrintToUser("AvalancheGo and Avalanche-CLI installed and node is bootstrapping!")
-	err = terraform.RemoveExistingTerraformFiles()
-	if err != nil {
-		return err
-	}
 	return nil
 }
 
 func requestAWSAccountAuth() error {
-	confirm := "Do you authorize Avalanche-CLI to access your AWS account to set-up your Avalanche Validator node?"
+	confirm := "Do you authorize Avalanche-CLI to access your AWS account to set-up your Avalanche Validator node? " +
```

Comment:
We can do this in a separate ticket if you want, but I'd like to list everything that command does before they accept here so they know what they're signing up for.